portfolio/src/main/webapp/index.html

```diff
@@ -1,19 +1,134 @@
 <!DOCTYPE html>
-<html>
-  <head>
-    <meta charset="UTF-8">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
-  </head>
-  <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click <a href="/hello">here</a> to see content created by server code.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-    </div>
-  </body>
-</html>
+<html lang="en">
+<head>
```

Comment:
(Optional) Certain optional tags, like <head> can be removed, for more details see https://google.github.io/styleguide/htmlcssguide.html#Optional_Tags.